src/zulip.rs

```diff
@@ -591,7 +581,8 @@ impl<'a> MessageApiRequest<'a> {
         self.recipient.url()
     }
 
-    pub async fn send(&self, client: &reqwest::Client) -> anyhow::Result<reqwest::Response> {
+    #[must_use]
+    pub async fn send(&self, client: &reqwest::Client) -> anyhow::Result<MessageApiResponse> {
```

Comment:
Isn't Future/Result `#[must_use]` by default?

Reply:
Yes, it is `#[must_use]`, but I though that if I added it it would also require usage of the inner T, but that doesn't seems to be the case. Removed them.

Reply:
No that's still a WIP feature/idea, IIRC :)